packages/teleterm/src/mainProcess/fixtures/mocks.ts

```diff
@@ -5,55 +5,66 @@ import { createMockFileStorage } from 'teleterm/services/fileStorage/fixtures/mo
 // Importing electron breaks the fixtures if that's done from within storybook.
 import { createConfigService } from 'teleterm/services/config/configService';
 
-const platform = 'darwin';
-
 export class MockMainProcessClient implements MainProcessClient {
+  configService: ReturnType<typeof createConfigService>;
+
+  constructor(private runtimeSettings: Partial<RuntimeSettings> = {}) {
+    this.configService = createConfigService(
+      createMockFileStorage(),
+      this.getRuntimeSettings().platform
+    );
+  }
   getRuntimeSettings(): RuntimeSettings {
-    return {
-      platform,
-      dev: true,
-      userDataDir: '',
-      binDir: '',
-      certsDir: '',
-      kubeConfigsDir: '',
-      defaultShell: '',
-      tshd: {
-        insecure: true,
-        requestedNetworkAddress: '',
-        binaryPath: '',
-        homeDir: '',
-        flags: [],
-      },
-      sharedProcess: {
-        requestedNetworkAddress: '',
-      },
-      tshdEvents: {
-        requestedNetworkAddress: '',
-      },
-      installationId: '123e4567-e89b-12d3-a456-426614174000',
-    };
+    return { ...defaultRuntimeSettings, ...this.runtimeSettings };
   }
 
   getResolvedChildProcessAddresses = () =>
     Promise.resolve({ tsh: '', shared: '' });
 
   openTerminalContextMenu() {}
-
   openClusterContextMenu() {}
-
   openTabContextMenu() {}
 
   showFileSaveDialog() {
     return Promise.resolve({ canceled: false, filePath: '' });
   }
 
-  configService = createConfigService(createMockFileStorage(), platform);
-
   fileStorage = createMockFileStorage();
 
   removeKubeConfig(): Promise<void> {
     return Promise.resolve(undefined);
   }
 
   forceFocusWindow() {}
+
+  async symlinkTsh() {
+    return true;
+  }
+  async removeTshSymlink() {
+    return true;
+  }
 }
+
+const defaultRuntimeSettings = {
+  platform: 'darwin' as const,
+  dev: true,
```

Comment:
> I know this is a mock, but what are the consequences of having our mock be in `dev` mode instead of not? Could this lead to any false positives/negatives?

`dev` is mostly used during the build & startup of the app, it shouldn't have much effect overall on tests.

In general I'm of the opinion that we shouldn't use `dev` for business logic, [I was talking with Grzegorz about this recently](https://github.com/gravitational/webapps/pull/1451#discussion_r1055385302). And for any other scenario, our setup code in tests should probably be closer to the dev build than the prod build (for example in terms of what's getting logged and what's not, etc.).